### DISCUSSION TITLE
This library breaks Lighthouse SEO score

### DISCUSSION THREAD

**Discussion by michenly** [google/model-viewer#3208] [General]
Doing Light house report on any page that uses `model-viewer` will cause the score to go down because this library uses an anchor link instead of a button to view AR
<img width="767" alt="Screen Shot 2022-02-16 at 11 55 41 AM" src="https://user-images.githubusercontent.com/1610169/154316531-ba12d5d2-bc6e-4c18-874f-be0a25866747.png">

Is this expected? I know un-styling all the buttons in different browser would be difficult, but maybe there is something that can be done in lighthouse? Does most people just ignore this score?

**Comment by elalish** [comment]
Thanks, I hadn't noticed that one. It may be a bug; I can't think a good reason for that button to be an anchor element.
